### ISSUE TITLE
Simplying Unlocking of app when Using E2E

### ISSUE THREAD

**Issue by p-sims** [johannes-schliephake/nextcloud-passwords-ios#287]
### Feature Description

Currently you have to unlock your device using passcode or TouchID/FaceID before you put in your E2E password. (if you choosing not to get iOS to save it).
Should there be an option to not require phone unlock if asking for E2E password to avoid having two stage unlocking process?

If this was possible then there would need to be a timeout that locked password app again if you switched apps for more than say 1 minute or 30 seconds. 

### Rationale

To avoid having two stage passcode entry process to unlock app. If you choose to require E2E password to be entered everytime, then you should be able to turn off the requirement for iOS authentification everytime you switch to the app.

**Comment by johannes-schliephake** [comment]
So the thing about FaceID / TouchID (and therefore the device's passcode) is that it's not just about the UI preventing you from accessing the app. Some parts of the encryption stack are relying on biometrics and I expect to use this mechanism even more in the future: FaceID will unlock the device's keychain (the framework, not the user facing feature) to retrieve the secrets needed to decrypt the entries stored for offline usage.

So in short there's no going around biometrics when launching the app. However, when the app is already running in the background and the secrets were already decrypted, I could add such a timeout and not instantly lock the app again. Would that help?

**Comment by p-sims** [comment]
> So in short there's no going around biometrics when launching the app. However, when the app is already running in the background and the secrets were already decrypted, I could add such a timeout and not instantly lock the app again. Would that help?

Yes, a time out would be an improvement. Prehaps we setting option to set time period. Thanks.
